15/root/usr/share/container-scripts/postgresql/common.sh

```diff
@@ -177,7 +177,7 @@ function generate_postgresql_recovery_config() {
 function generate_passwd_file() {
   export USER_ID=$(id -u)
   export GROUP_ID=$(id -g)
-  grep -v -e ^postgres -e ^$USER_ID /etc/passwd > "$HOME/passwd"
+  grep -v -e ^postgres -e ^$USER_ID -e ^$(id -un) /etc/passwd > "$HOME/passwd"
```

Comment:
We are there mixing `^postgres` `^$USER_ID` and `^$(id -un)`. What about to add there `local USERNAME=$(id -un)` so we do not mix three type of grep expressions. Only nit-picking.

And does it mean, that functionality without `userns=keep-id` works as well with this change?

Reply:
We have `USER_ID` and `GROUP_ID` variables because the values are used in multiple places so it makes sense there. The username is only used in the `grep` call, so I would rather not introduce a variable for it.

>And does it mean, that functionality without userns=keep-id works as well with this change?

It should, yes. The tests are green after all. :)